services/eth_gateway_test.go

```diff
@@ -234,11 +234,7 @@ func Test_sendEth(t *testing.T) {
 	transferValue := big.NewInt(1)
 	transferValueInWei := new(big.Int).Mul(transferValue, oneWei)
 	// Send ether to test account
-
-	prlWallet := getWallet(prl1File)
-
-	txs, _, _, err := services.EthWrapper.SendETH(prlWallet.Address,
-		prlWallet.PrivateKey, ethAddress02, transferValueInWei)
+	txs, _, _, err := services.EthWrapper.SendETH(services.MainWalletAddress, services.MainWalletPrivateKey, ethAddress02, transferValueInWei)
```

Comment:
Did you use the MainWalletAddress in the unit tests on purpose?

Reply:
You're correct, I have replaced with the prlWallet.